src/views/organisations/Edit.vue

```diff
@@ -91,10 +95,10 @@ export default {
           if (data.url === this.organisation.url) {
             delete data.url;
           }
```

Comment:
Support now nullable value.

```suggestion
          if (data.url === (this.organisation.url || "")) {
            delete data.url;
          }
```

Reply:
Done